config/initializers/subscribers.rb

```diff
@@ -0,0 +1,5 @@
+# This initializer allows to register subscribers
+
+ActiveSupport::Notifications.tap do |config|
+  # config.subscribe :my_event, MySubscriber
+end
```

Comment:
Is this a leftover? I'd encourage engs to write plugins if they want to subscribe. Core should only provide minimum contract I think.